Nodes/config/routes.php

```diff
@@ -14,17 +15,6 @@
     $routeBuilder->connect('/promoted/*', ['controller' => 'Nodes', 'action' => 'promoted']);
     $routeBuilder->connect('/search/*', ['controller' => 'Nodes', 'action' => 'search']);
 
-    $routeBuilder->scope('/:type', function (RouteBuilder $routeBuilder) {
-        $routeBuilder->routeClass('Croogo/Taxonomy.TypeRoute');
-        $routeBuilder->connect('/', ['controller' => 'Nodes', 'action' => 'index']);
-        $routeBuilder->connect('/archives/*', ['controller' => 'Nodes', 'action' => 'index']);
-        $routeBuilder->connect('/:slug', ['controller' => 'Nodes', 'action' => 'view']);
-        $routeBuilder->connect('/term/:slug/*', ['controller' => 'Nodes', 'action' => 'term']);
-    });
-
     // Content types
-    $routeBuilder->connect('/node', ['controller' => 'Nodes', 'action' => 'index']);
-    $routeBuilder->connect('/node/archives/*', ['controller' => 'Nodes', 'action' => 'index']);
-    $routeBuilder->connect('/node/:slug', ['controller' => 'Nodes', 'action' => 'view']);
-    $routeBuilder->connect('/node/term/:slug/*', ['controller' => 'Nodes', 'action' => 'term']);
+    CroogoRouter::routableContentTypes($routeBuilder);
```

Comment:
https://github.com/croogo/croogo/blob/3.0/Taxonomy/src/Routing/Route/TypeRoute.php already does this for all Types, without needing to make `Croogo/Core` dependant on `Croogo/Taxonomy`. Only thing is to perhaps add the `routes=true` param check?

I also don't know if it's a good idea to remove the generic/catch-all node route?